gdnative-derive/src/native_script/property_args.rs

```diff
@@ -102,6 +105,25 @@ impl PropertyAttrArgsBuilder {
                     ));
                 }
             }
+            "get" => {
+                let string = if let syn::Lit::Str(lit_str) = &pair.lit {
+                    lit_str.value()
+                } else {
+                    return Err(syn::Error::new(
+                        pair.span(),
+                        "get value is not a string literal".to_string(),
+                    ));
+                };
+
+                let path =
+                    syn::parse_str::<syn::Path>(string.as_str()).map_err(invalid_value_path)?;
+                if let Some(old) = self.get.replace(path) {
+                    return Err(syn::Error::new(
+                        pair.span(),
+                        format!("there is already a get value set: {:?}", old),
```

Comment:
"get value set" is a bit confusing, maybe "there is already a 'get' attribute with value:"

same below

Reply:
Agree. Actually I copy-paste this from other arms inside the same method (`add_pair`) and others has almost the same format of error message. How about use a pair of back quote (``"`"``) to surround the variable name?
For example, ``"there is already a `get` value set:\n{:?}"``. I think this will be more convenient to write a macro to generate each arm? (But I am not familiar with macro, not sure whether it is suitable to use in this case)

Reply:
You can also use backticks, they will likely not be formatted though.

Why a macro? Not sure I understand it...

Reply:
Because every arm (e.g. `"set"`, `"get"`) has a very similar implementation, only differs in field name. I think it will be better if these arms can be generated by macro. (Like how this crate implement `Export` trait for core types ([source](https://github.com/godot-rust/godot-rust/blob/master/gdnative-core/src/export/property.rs#L325-L484)))